app/views/layouts/application.html.erb

```diff
@@ -7,9 +7,40 @@
 
     <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
+    <link href="https://blackrockdigital.github.io/startbootstrap-bare/vendor/bootstrap/css/bootstrap.min.css" rel="stylesheet">
```

Comment:
avoid using CDNs, import bootstrap in application.scss

Reply:
okay sir, I'll make the necessary change.

Reply:
I've made the required change